apps/browser/qml/pages/components/ToolBar.qml

```diff
@@ -124,17 +124,6 @@ Column {
         }
     }
 
-    SecondaryBar {
-        id: secondaryBar
-        visible: opacity > 0.0 || height > 0.0
-        opacity: secondaryToolsActive ? 1.0 : 0.0
-        height: secondaryToolsHeight
-        horizontalOffset: toolBarRow.horizontalOffset
-        iconWidth: toolBarRow.iconWidth
-
-        Behavior on opacity { FadeAnimation {} }
-    }
-
```

Comment:
There are a bunch of animations on the toolbar that no longer seem relevant with the new pop up menu. For example, it doesn't seem to make sense to hide the back button (line 197) or padlock icon (line 207). The animations on the reload (line 328) and menu (lines 349 and 358) buttons are also no longer visible.

Reply:
What's the status of this?

Reply:
done

Reply:
Thank you, it looks much cleaner now.